doc/source/cluster/kubernetes/user-guides/persist-kuberay-custom-resource-logs.md

```diff
@@ -191,14 +191,15 @@ kind: RayCluster
 metadata:
   name: raycluster-fluentbit-sidecar-logs
 spec:
-  rayVersion: '2.9.0'
+  rayVersion: '2.46.0'
   headGroupSpec:
-    rayStartParams: {}
+    # Before KubeRay 1.4.0, rayStartParams is required. It's optional after that.
+    # rayStartParams: {}
     template:
       spec:
         containers:
         - name: ray-head
-          image: rayproject/ray:2.9.0
+          image: rayproject/ray:2.47.1
```

Comment:
```suggestion
          image: rayproject/ray:2.46.0
```

This should also be 2.46.0. Ditto for other files.

Reply:
whoops, forgot. Updated all, thanks